mulighetsrommet-api/src/main/resources/db/migration/V123__tiltakstype_arena_kode.sql

```diff
@@ -0,0 +1,16 @@
+alter table tiltakstype rename column tiltakskode to arena_kode;
+alter table tiltakstype add column tiltakskode text;
+alter table tiltakstype drop column skal_migreres;
```

Comment:
Kunne valgt et annet navn for tiltakskode for å skille mer fra det vi "tidligere" kalte tiltakskode? F. eks navn (og så ble navn -> arenaNavn).

arena_kode kunne også potensielt hett tiltakskodeArena som team tiltak kaller det.